configuration/configuration_test.go

```diff
@@ -89,9 +89,10 @@ var configStruct = Configuration{
 			MinimumTLS   string   `yaml:"minimumtls,omitempty"`
 			CipherSuites []string `yaml:"ciphersuites,omitempty"`
 			LetsEncrypt  struct {
-				CacheFile string   `yaml:"cachefile,omitempty"`
-				Email     string   `yaml:"email,omitempty"`
-				Hosts     []string `yaml:"hosts,omitempty"`
+				CacheFile    string   `yaml:"cachefile,omitempty"`
+				Email        string   `yaml:"email,omitempty"`
+				Hosts        []string `yaml:"hosts,omitempty"`
+				DirectoryURL string   `yaml:"directoryurl,omitempty"`
```

Comment:
Not new in this PR, but I realise how awkward that `Configuration` type is to use, because it looks like the only way to construct a literal is to, well, literally duplicate the `Configuration` definition, including all the `yaml:` labels 🙈  We should probably have a look at that.

Reply:
cc: @lavalleeale same here